draft-ietf-quic-transport.md

```diff
@@ -5277,15 +5277,16 @@ Largest Acknowledged:
 
 ACK Delay:
 
-: A variable-length integer representing the time delta in microseconds between
-  when the ACK frame was sent and when the largest acknowledged packet, as
-  indicated in the Largest Acknowledged field, was received by this peer.  The
-  value of the ACK Delay field is scaled by multiplying the encoded value by 2
-  to the power of the value of the ack_delay_exponent transport parameter set by
-  the sender of the ACK frame; see {{transport-parameter-definitions}}.  Scaling
-  in this fashion allows for a larger range of values with a shorter encoding at
-  the cost of lower resolution.  Because the receiver doesn't use the ACK Delay
-  for Initial and Handshake packets, a sender SHOULD send a value of 0.
+: A variable-length integer encoding the intentional delay between when the
+  largest acknowledged packet, as indicated in the Largest Acknowledged field,
+  was received, and when the ACK frame was sent; see {{host-delay}}.
+  The acknowledgement delay in microseconds is the value of the ACK Delay
+  field multiplied by 2 to the power of the ack_delay_exponent transport
+  parameter sent by the sender of the ACK frame;
+  see {{transport-parameter-definitions}}.  Scaling in this fashion allows for
+  a larger range of values with a shorter encoding at the cost of lower
+  resolution.  Because the receiver doesn't use the ACK Delay for Initial and
+  Handshake packets, a sender SHOULD send a value of 0.
```

Comment:
```suggestion
: A variable-length integer encoding the acknowledgement delay in
  microseconds; see {{host-delay}}. It is decoded by multiplying the
  value in the field by 2 to the power of the ack_delay_exponent transport
  parameter sent by the sender of the ACK frame; see
  {{transport-parameter-definitions}}.  This encoding allows for a large
  range of values, although it sacrifices some resolution.
  
  Since the acknowledgement delay is not used for Initial and Handshake
  packets, the ACK Delay field in acknowledgements for those packet types
  SHOULD be set to 0.
```

Also, I would move this last paragraph up to the host-delay section.

Reply:
The description of the delay is already in the host delay section. We can fix that section if necessary, but let's not repeat the same text down here.

Reply:
That last bit won't format correctly.  You need a ': ' at the start of the line, or maybe we could move it elsewhere.

Reply:
I think it can be moved up to the host delay section.

Reply:
Agreed, I'll move the last bit down to the host delay section.

Reply:
Thanks, I kept most of the existing sentence about size vs resolution, but otherwise took your suggestion @janaiyengar